### DISCUSSION TITLE
Update default postal settings

### DISCUSSION THREAD

**Discussion by freelovehere** [postalserver/postal#2961] [Help with using Postal]
I saw somewhere that some certain areas of postal can be changed and I’d love to do that myself. I am not a guru but I’d love to be pointed the right direction to make this update. I have attached pictures demonstrating what I am talking about. Thank you 
![d83dfa08-b910-4d59-a047-6d32edd9f2f2](https://github.com/postalserver/postal/assets/169105514/37ab6793-c200-46b8-9d96-c97d0432e223)
![344e04ef-907e-4c3b-b4dc-b4c89058a86e](https://github.com/postalserver/postal/assets/169105514/69cf41e0-5791-462b-bbfa-92ead0370e46)
![3c885299-be0b-4fb0-bd5b-aba36a70e890](https://github.com/postalserver/postal/assets/169105514/07c5d3e2-45a6-4e0e-9acc-19be5478c4bf)

**Comment by willpower232** [comment]
https://github.com/postalserver/postal/blob/d5104991903d42563c6c6f7ff01276d496e3304f/app/views/messages/activity.html.haml#L40

https://github.com/postalserver/postal/blob/d5104991903d42563c6c6f7ff01276d496e3304f/app/controllers/messages_controller.rb#L23

You would have to use docker compose mounts to replace the files or create your own Docker image.

**Reply by freelovehere** [reply]
Thank you @willpower232 , as I said earlier, I am not a guru or an expert in these things, I would appreciate it if I can get the code to just copy and paste or more give a little more details on how to use the docker compose to fix this. Thank you once more
